### DISCUSSION TITLE
Azure DevOps Extension loads all chunks but doesn't display a report in pipeline runs

### DISCUSSION THREAD

**Discussion by jreimone** [allure-framework/allure2#2822] [General Discussion]
For a few weeks now I noticed that reports are not displayed. I only see the chunks being loaded but afterward nothing happens.
![Screenshot 2024-12-09 at 13 51 08](https://github.com/user-attachments/assets/3b4b1d50-cdc9-4f05-827d-9a6904a2ad09)

After the chunks got loaded I get the following error in the console:
![Screenshot 2024-12-09 at 13 52 35](https://github.com/user-attachments/assets/fc421c5f-af20-44a7-b813-2104c87cb5b7)

Please advise, how to fix that.

**Comment by baev** [comment]
duplicate of https://github.com/orgs/allure-framework/discussions/2723#discussion-7164864
